packages/prompts/src/index.ts

```diff
@@ -624,3 +624,46 @@ Only return Node.js code. DO NOT include any English text before or after the No
 
 `;
 };
+
+export const englishStyleGuidePrompt = `# English Style Guide
+
+## Instructions
+
+Your goal is to apply two style guides to any English you find in the code:
+
+1. Google's Developer docs style guide (https://developers.google.com/style)
+2. Dreyer's English
+
+You might see English in places like the following (this is not an exhaustive list):
+
+- Code comments
+- Code strings
+- Any string you find in the code (e.g. the values of properties, variables, etc.)
+- Markdown (headings, text, lists, etc.)
+- HTML attributes, like img alt tags
+- etc.
+
+Please apply the rules of the style guide to ANY English you find.
+
+## Input format
+
+Below, I'm passing data as a CSV of the following format:
+
+- \`key\`: A string, a composite key of the following format: \`<file path>:<line number>\`
+- \`code\`: A string, the code found at the \`key\` location
+
+## Output format
+
+You MUST ONLY output your results as a CSV of the following format:
```

Comment:
```suggestion
You must only output your results as a CSV of the following format:
```

Reply:
Changed MUST ONLY to must only for consistency with Google's Developer docs style guide